### PR TITLE
Scanner update

### PR DESCRIPTION
**Pull request by yarunachalam**
Included NeuVector Scanner Image Update section

### CODE REVIEW

**Review by jeremy-moffitt — COMMENTED** [review]
This looks ok, but lets wait until we have a final decision on the scanner update to commit it since we may change

File: content/neuvector-prime/azure/contents.lr
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: I would include something above this that explains why the customer might want to turn this on... Maybe "Upgrading your NeuVector offering requires the use of the Azure automatic upgrade functionality. This will allow NeuVector to regularly update its database of threat definitions."

File: content/neuvector-prime/azure/contents.lr
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: Maybe add "To manually control the timing of upgrade, "  before the "You have an option..."

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
some minor tweaks, generally looks good

File: content/neuvector-prime/azure/contents.lr
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: This needs more  detail. When you first enable it , it will show updating but that just means updating from non-auto to auto to vice versa depending on the current state and previous state.  Once it is enabled and it detects an update is available it will show "Updating" again but this time you will see the version has changed.  If they enable auto upgrade and they are on the latest, the would not see this second updating.

**Review by jeremy-moffitt — APPROVED** [review]

File: content/neuvector-prime/azure/contents.lr
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: With the latest decision, auto update is enabled by default. Section is removed.

**Review by jeremy-moffitt — APPROVED** [review]
LGTM, thanks for making the auto update updates

File: content/neuvector-prime/azure/contents.lr
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: s/Enabled/enabled

File: content/neuvector-prime/azure/contents.lr
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: s/te/the

File: content/neuvector-prime/azure/contents.lr
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: By default, the Kubernetes Service Extension property "Auto upgrade minor version" is Enabled during the NeuVector offer deployment. This will allow NeuVector to regularly update its CVE database.

**Review by KeithMnemonic — CHANGES_REQUESTED** [review]
a few minor tweaks/typos

File: content/neuvector-prime/azure/contents.lr
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: With Auto upgrade minor version" enabled, the deployment should always be on the latest version. To see the 3 most recent published versions, Execute the following command in Cloud Shell

Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: done

File: content/neuvector-prime/azure/contents.lr
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: By default the Kubernetes Service Extension property "Auto upgrade minor version" is enabled during the NeuVector offer deployment. This will allow NeuVector to regularly update its CVE database.

Click "Refresh" in the Extension + application screen to view the latest Version.

File: content/neuvector-prime/azure/contents.lr
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: done

**Pull request merged**
